portal/static/js/src/modules/TnthAjax.js

```diff
@@ -272,6 +272,60 @@ export default { /*global $ */
             }
         });
     },
+    "getOrg": function(orgId, params, callback) {
+        callback = callback || function() {};
+        if (!orgId) {
+            callback({error: i18next.t("Organization id is required.")});
+            return false;
+        }
+        if (sessionStorage[`orgData_${orgId}`]) {
+            callback(JSON.parse(sessionStorage[`orgData_${orgId}`]));
+            return true;
+        }
+        params = params || {};
+        /* individual org */
+        this.sendRequest("/api/organization/"+orgId, "GET", "", params, function(data) {
+            if (!data.error) {
+                $(".get-orgs-error").html("");
+                sessionStorage.setItem(`orgData_${orgId}`, JSON.stringify(data));
+                callback(data);
+            } else {
+                var errorMessage = i18next.t("Server error occurred retrieving organization/clinic information.");
+                $(".get-orgs-error").html(errorMessage);
+                callback({"error": errorMessage});
+            }
+        });
+    },
+    "getResearchStudies": function(userId, params, callback) {
+        callback = callback || function() {};
+        if (!userId) {
+            callback({error: i18next.t("User id is required.")});
+            return false;
+        }
+        this.sendRequest("/api/user/" + userId + "/research_study", "GET", userId, params, function(data) {
+            if (data) {
+                if (!data.error) {
+                    callback(data);
+                    return true;
+                } else {
+                    callback({"error": i18next.t("Server error occurred retrieving research study information.")});
```

Comment:
Are you able to reuse an existing "server error" error string here? I'd prefer not to have a error message (and 10+ translations) for every API we implement